### ISSUE TITLE
Can you save games at all or not?

### ISSUE THREAD

**Issue by AktiveDekay** [Vita3K/Vita3K#3460]
So, I recently got Vita3K and I wanted to play some nostalgic visual novels (Diabolik Lovers, Hakuoki, Amnesia, etc), but I know for a fact none of my Diabolik Lovers games are saving at all. I can go to the menu and try to save a game file, but nothing happens and an error pops up in the control panel for the emulator. 

I see conflicting reports about being able to save games, and I was just wondering if it's a bug or if it's a feature to be added, or what? Can I save at all? Is there a way to implement a save state so I don't have to lose all my progress when I reboot the games?

**Comment by Slade1972** [comment]
Yes, you can save games. Most games save correctly. So far, there are only a small number of tested games that do not save.

There are 6 games listed under Diabolik Lovers... I'll just pick one at random and hope for the best.

**Comment by AktiveDekay** [comment]
> Yes, you can save games. Most games save correctly. So far, there are only a small number of tested games that do not save.
> 
> There are 6 games listed under Diabolik Lovers... I'll just pick one at random and hope for the best.

I mean... I tried saving, but it didn't. Haunted Dark Bridal and MORE,BLOOD don't save at all. I've tried repeatedly to save these games, because there's certain things locked if you don't make it through an entire route, and they just... Wouldn't save.

**Comment by Slade1972** [comment]
> I mean... I tried saving, but it didn't. Haunted Dark Bridal and MORE,BLOOD don't save at all. I've tried repeatedly to save these games, because there's certain things locked if you don't make it through an entire route, and they just... Wouldn't save.

There was an issue I found with some visual novel the other day, same thing, the game wouldn't save. I posted the issue (in the compatibility database) and @Zangetsu38 fixed it within a day or so. That fix more than likely fixed the issue with these games as well. They all seem to use the exact same game engine.

Update Vita3k and you should be good to go now. At least on some of these games.  Just to make sure it worked, I saved, quit, then reloaded and the save worked fine.

You might have to give me the Title ID codes for those you've just mentioned, I can't seem to find them.

![dllv1](https://github.com/user-attachments/assets/e5353e41-58ea-4d12-b60e-8a5af2a8fbdf)

**Comment by Zangetsu38** [comment]
Oh, i have fixed very lot more game i thinks lol

**Comment by scribam** [comment]
@AktiveDekay can you update Vita3K and check if you still have the issue?

**Comment by Zangetsu38** [comment]
reena said me have never get problem for save on this game, an i remmeber my quick test same

**Comment by Dicot0721** [comment]
> PCSG00511 Wizardry 巫術_囚禁亡靈的街道
> PCSG00512 Wizardry 巫術_被囚之魂的迷宮
> 進入存檔畫面後，不能正常顯示，也無法操作。
> 
> After entering the save screen, it cannot be displayed properly and cannot be operated.

Please use the latest version of the emulator to test.

**Comment by nishinji** [comment]
Outdated
